nbconvert/templates/html/slides_reveal.tpl

```diff
@@ -211,13 +209,29 @@ require(
 
         Reveal.addEventListener('slidechanged', update);
 
-        var update_scroll = function(event){
-          $(".reveal").scrollTop(0);
-        };
-
-        Reveal.addEventListener('slidechanged', update_scroll);
+        function setScrollingSlide(scroll) {
+
+            if (scroll === "True") {
+              var h = $('.reveal').height() * 0.95;
+              var hpx = "" + h + "px";
+              $('section.present').find('section')
+                .filter(function() {
+                  return $(this).height() > h;
+                })
+                .css('height', hpx)
```

Comment:
> Does it work to use a height like 95%

Nop, it cut the stuff at the very top.

> even CSS calc

Neither... in both cases I get the cut off

![slide2](https://cloud.githubusercontent.com/assets/1640669/26698564/d8a3202e-46ec-11e7-97e2-dbba903bbe8a.png)

It seems I need to specify the the absolute height in pixel to make it work.